roles/get-requirements.yaml

```diff
@@ -172,6 +172,23 @@
           register: "output"
           when: not helm_secrets_exists.stat.exists or (helm_secrets_removed.changed)
 
+    - name: azure-cli
+      block:
+        - name: Check if azure-cli exists
+          stat:
+            path: "/usr/bin/az"
+          register: azure_cli_exists
+
+        - name: Check azure-cli
+          shell: az version --query '"azure-cli"'
+          register: current_azure_cli_version
+          when: azure_cli_exists.stat.exists
+
+        - name: Get azure-cli
+          command: "curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash"
+          when: (not azure_cli_exists.stat.exists) or
+            (current_azure_cli_version.stdout == "")
```

Comment:
Is there a more predictable installation method? I'd like to be able to determine which version is getting installed.

I think you can install it via pip?

Reply:
Won't that make pip a requirement then?

Reply:
Yea that is unfortunate. Is it available in apt repositories so that you can add it to `apt_list`?

Reply:
Yes, but it needs to add the Microsoft repo for it. I'll look into that.